src/AccessibilityInsights/MainWindowHelpers/ServerSignIn.cs

```diff
@@ -0,0 +1,149 @@
+// Copyright (c) Microsoft. All rights reserved.
```

Comment:
could you explain the relevance of this change in this PR? I wonder why we need this change in this PR.

Reply:
I believe this class was unintentionally added